assets/scripts/gameboardlogic.js

```diff
@@ -111,20 +113,21 @@ const flipMark = function () {
 
 // Adds the cards to the DOM.
 const setUpGameBoardHandlers = function (firstTime) {
-  console.log('setUpGameBoard')
-  $('#cell-0-img').one('click', flipMark)
-  $('#cell-1-img').one('click', flipMark)
-  $('#cell-2-img').one('click', flipMark)
-  $('#cell-3-img').one('click', flipMark)
-  $('#cell-4-img').one('click', flipMark)
-  $('#cell-5-img').one('click', flipMark)
-  $('#cell-6-img').one('click', flipMark)
-  $('#cell-7-img').one('click', flipMark)
-  $('#cell-8-img').one('click', flipMark)
+  for (let i = 0; i < gameCellIDs.length; i++) {
```

Comment:
Yes.  I'm new to javascript and struggled with with id's and jquery. Thanks for your insight.